nuc_morph_analysis/analyses/linear_regression/supplemental_lrm_figure_workflow.py

```diff
@@ -0,0 +1,63 @@
+#%%
+import numpy as np
+from nuc_morph_analysis.lib.preprocessing import global_dataset_filtering, filter_data
+from nuc_morph_analysis.analyses.linear_regression.linear_regression import fit_linear_regression
+from nuc_morph_analysis.analyses.linear_regression.analysis_plots import (run_regression_workflow,
+                                                                          plot_feature_cluster_correlations, 
+                                                                          plot_heatmap, 
+                                                                          plot_feature_contribution)
+from nuc_morph_analysis.analyses.linear_regression.select_features import (get_feature_list, 
+                                                                           TARGET_SETTINGS)
+
+#%%
+df_all = global_dataset_filtering.load_dataset_with_features()
+df_full = filter_data.all_timepoints_full_tracks(df_all)
+df_track_level_features = filter_data.track_level_features(df_full)
+
+FIGDIR='linear_regression/figures/'
+TARGETS = ['duration_BC', 'delta_volume_BC']
+CONFIG = {
+    'all_features': ['start_intrinsic', 'lifetime_intrinsic', 'start_extrinsic', 'lifetime_extrinsic'],
+    'start_intrinsic': ['start_intrinsic'],
+    'lifetime_intrinsic': ['lifetime_intrinsic'],
+    'start_extrinsic': ['start_extrinsic'],
+    'lifetime_extrinsic': ['lifetime_extrinsic'],
+    'intrinsic': ['start_intrinsic', 'lifetime_intrinsic'],
+    'extrinsic': ['start_extrinsic', 'lifetime_extrinsic'],
+}
+EXTENDED_WORKFLOW = False
```

Comment:
what is extended?

Reply:
I am going to reformat the workflow to make this more clear actually.. please hold :)

Reply:
Okay! I updated the workflow to re-compute the max alpha and use that for the feature importance plot. This takes ~20 minutes because it has to sweep through a range of alphas linearly and added volume as a high max alpha. 

If you change the RECOMPUTE_ALPHA to False it will use pre-computed values and run faster. 

Thanks for asking!